hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamerSchemaEvolutionExtensive.java

```diff
@@ -195,7 +195,7 @@ protected void testBase(String updateFile, String updateColumn, Map<String,Integ
   protected static Stream<Arguments> testArgs() {
     Stream.Builder<Arguments> b = Stream.builder();
     //only testing row-writer enabled for now
-    for (Boolean rowWriterEnable : new Boolean[]{true}) {
+    for (Boolean rowWriterEnable : new Boolean[]{false, true}) {
```

Comment:
Could you file a ticket to reduce the combinations here? by replacing that with light-weight tests.